packages/testlab/test/unit/misc.test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '../../src/expect';
+import {toJSON} from '../../src/misc';
+
+describe('toJSON', () => {
```

Comment:
Please add the following test cases:
- class Customer: a customer instance => plain json
- class Customer and Address: a customer with address instance => plain json
- Arrays (top level or embedded)
- A function, such as `function x() {}` - I think it will throw `SyntaxError: Unexpected token u in JSON at position 0` as `JSON.stringify(x)` produces `undefined`.

Reply:
> class Customer: a customer instance => plain json
> class Customer and Address: a customer with address instance => plain json

So far, `@loopback/testlab` does not depend on `@loopback/repository`. Are we ok with adding this dependency?

> Arrays (top level or embedded)

Makes sense 👍 

> A function, such as function x() {}

I think this will be rejected by TypeScript compiler, but I can add a check for this to accommodate for people calling this API from JavaScript.